_clients/logstash/read-from-opensearch.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Read from OpenSearch
+parent: Logstash
+nav_order: 220
+---
+
+# Read from OpenSearch
+
+As we ship Logstash events to an OpenSearch cluster using the OpenSearch output plugin. We can also perform read operations on an OpenSearch cluster and load data into Logstash using the OpenSearch input plugin.
+
+The OpenSearch input plugin helps to read the search query results performed on an OpenSearch cluster and load them into Logstash. This is useful for replaying test logs, reindexing, etc. Users can schedule this ingestion to run periodically using 
+cron syntax(using `schedule` configuration setting) or by just running the query once to load data into Logstash.
```

Comment:
"The OpenSearch input plugin reads the search query results performed on an OpenSearch cluster and loads them into Logstash. This lets you replay test logs, reindex, and perform other operations based on the loaded data. You can schedule ingestions to run periodically by using [cron expressions]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron), or manually load data into Logstash by running the query once.

Reply:
ack